qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -206,6 +207,16 @@ def _param_setter(self, set_cmd: str, value: Union[float, str]) -> None:
         #  the intended value
 
 
+class MercurySlavePS(MercuryWorkerPS):
+    """
+    Deprecated class
+    """
+    def __init__(self, parent: VisaInstrument, name: str, UID: str) -> None:
+        issue_deprecation_warning("MercurySlavePS class", "",
```

Comment:
i believe that `deprecate` decorator works on classes, so this one can be just made into a `pass` class.